packages/cosmic-swingset/test/provision-smartwallet.test.ts

```diff
@@ -99,6 +110,7 @@ test.skip('integration test: smart wallet provision', async t => {
   ]);
 
   t.log('verify 10BLD spent, 0.25 IST received');
+  // Assuming queryBalance returns an object with a balances property
   t.deepEqual(addrQ.balances, [
```

Comment:
Obvious from the following line.
```suggestion
  t.deepEqual(addrQ.balances, [
```